### DISCUSSION TITLE
umijs文件的请求非常慢

### DISCUSSION THREAD

**Discussion by kuangshu** [umijs/umi#11435] [Q&A] [answered]
使用版本：
4.0.72

第一次启动的 umijs 加载非常慢
![image](https://github.com/umijs/umi/assets/21115683/553932cc-42a5-45e6-a3fb-ccd7414b3a8f)
1.2m 的文件可以达到惊人的 4m 

回退到升级前的 4.0.15 发现没有这个问题
![image](https://github.com/umijs/umi/assets/21115683/ee5e0a05-e801-4682-9595-10a9abf67f5a)

我看了下 express 只差了个小版本，4.18.1 到 4.18.2，好像 webpack-dev-middleware 从 5.3.3 升级到了 6.0.1，不知道是不是这个升级产生的影响。
这种问题应该如何排查？影响可能来自那里？

**Comment by fz6m** [comment]
看一下你的控制台有没有开启 disable cache 或者 throttling 限速，如有问题，需要给一个完整复现的视频帮助排查问题。

**Reply by kuangshu** [reply]
今天我初步排查了下插件里面的配置，发现关掉 `forkTSChecker`中的 `async` 就是正常的。
之前自己的插件里面有如下配置：
```
 config.forkTSChecker = {
    async: isDev
  };
```

现在先决定不升级了。升级的问题之后有空了再排查。

**Reply by fz6m** [reply, marked as answer]
看一下这个对你有没有帮助：#10935
